docs/llm_serving.md

```diff
@@ -0,0 +1,278 @@
+# Efficient LLM Serving {#ovms_docs_llm_serving}
+
+**THIS IS A PREVIEW FEATURE**
+
+## Overview
+
+With rapid development of generative AI, new techniques and algorithms for performance optimization and better resource utilization are introduced to make best use of the hardware and provide best generation performance. OpenVINO implements those state of the art methods in it's [GenAI Library](https://github.com/ilya-lavrenov/openvino.genai/tree/ct-beam-search/text_generation/causal_lm/cpp/continuous_batching/library) like:
+  - Continuous Batching
+  - Paged Attention
+  - Dynamic Split Fuse 
+  - *and more...*
+
+ It is now integrated into OpenVINO Model Server providing efficient way to run generative workloads.
+
+## Quickstart
```

Comment:
I suggest a separate page with a quick start guide. It should use fairly small model without authorization like tinyllama.
It should be very simple and short.